src/store.js

```diff
@@ -166,6 +166,10 @@ class AsyncStorageStore {
   async resetWallet() {
     // This should delete the access data and registered tokens, the only persisted data
     await AsyncStorage.multiRemove(walletKeys);
+    // Delete the same keys on the memory cache
```

Comment:
I feel the `await this.clearItems(true);` line below is already deleting those keys, isn't it?

Reply:
Actually it's not but it's confusing. `onlyWalletKeys` and `walletKeys` don't mean the same thing

Reply:
Yes, onlyWalletKeys mean the legacy wallet keys which are prefixed with wallet: